ui/src/pages/version/components/forms/components/transformer/InputPanel.js

```diff
@@ -170,7 +184,18 @@ export const InputPanel = ({ inputs = [], onChangeHandler, errors = {} }) => {
                   ])
                 }
               />
-            </EuiFlexItem>
+              </EuiFlexItem>
+              { protocol === PROTOCOL.UPI_V1 && (
+                <EuiFlexItem>
+                  <AddButton
+                    title="+ Add Autoload"
+                    description="Autoload configuration register tables and variables to standard transformer registry"
```

Comment:
Replace with
`UPI Autoload configuration register tables and variables in UPI request payload into standard transformer registry`